### PR TITLE
AN-504: Fixed missing httpGateway section in quick-deploy-gcp config.json

### PR DESCRIPTION
**Pull request by martinkolenic**
https://api3dao.atlassian.net/browse/AN-504

### CODE REVIEW

File: docs/airnode/v0.4/grp-providers/tutorial/quick-deploy-gcp/src/quick-deploy-gcp/config/config.json [hunk 1]
```diff
@@ -33,6 +33,9 @@
     "heartbeat": {
       "enabled": false
     },
+    "httpGateway": {
```
Comment (Siegrift) [inline docs/airnode/v0.4/grp-providers/tutorial/quick-deploy-gcp/src/quick-deploy-gcp/config/config.json:36]: Isn't the gateway optional?

Comment (amarthadan) [reply docs/airnode/v0.4/grp-providers/tutorial/quick-deploy-gcp/src/quick-deploy-gcp/config/config.json:36]: You still need to have the field in the config even if it's disabled.

Comment (wkande) [reply docs/airnode/v0.4/grp-providers/tutorial/quick-deploy-gcp/src/quick-deploy-gcp/config/config.json:36]: So this is a (required) field now? If so I will change it **[here](https://docs.api3.org/airnode/v0.4/reference/deployment-files/config-json.html#httpgateway)** later..

Comment (wkande) [reply docs/airnode/v0.4/grp-providers/tutorial/quick-deploy-gcp/src/quick-deploy-gcp/config/config.json:36]: I figured it out, it seems to be required. Update the config.md file in Deployment files. Add a little clarity on the boolean.

**Review by wkande — APPROVED** [review]
I made a few very small changes to reflect that the field is required.

**Pull request merged**